docs/_sass/_breakpoints.scss

```diff
@@ -0,0 +1,9 @@
+$breakpoint-tablet: 800px;
+$breakpoint-desktop: 1120px;
+$breakpoint-wide: 1600px;
```

Comment:
NAB, naming them `$breakpoint-x` feels slightly redundant to me, I'd probably just go with `$tablet` or `$tablet-min`

Reply:
I just don't want the variables to potentially conflict with other variables in the future, since we are not using the modern sass module system with `@use` in here.

Reply:
If this were a proper module rather than just a partial I'd agree

Reply:
Also this is more consistent with what we have in other sass partials (i.e: `$color-blue` instead of just `$blue`)

Reply:
Makes sense 👍